layouts/account.vue

```diff
@@ -0,0 +1,109 @@
+<script setup lang="ts">
+import { storeToRefs } from 'pinia';
+import { get } from '@vueuse/core';
+import { useMainStore } from '~/store';
+import { commonAttrs, noIndex } from '~/utils/metadata';
+import Default from '~/layouts/default.vue';
+
+const { t } = useI18n();
+
+const store = useMainStore();
+const { account } = storeToRefs(store);
+
+const username = computed(() => get(account)?.username);
+
+useHead({
+  title: 'account',
+  meta: [
+    {
+      key: 'description',
+      name: 'description',
+      content: 'Manage your rotki premium account',
+    },
+    noIndex(),
+  ],
+  ...commonAttrs(),
+});
+
+useAutoLogout();
+const tabModelValue = ref();
+</script>
+
+<template>
+  <Default>
```

Comment:
is this the only way to use a layout in another layout?

Reply:
we can use `<NuxtLayout>`